src/client/styles/Modal.css

```diff
@@ -0,0 +1,61 @@
+.modal-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1040;
+  width: 100vw;
+  height: 100vh;
+  background-color: #000;
+  opacity: 0.5;
+}
+
+.modal-wrapper {
+  position: fixed;
+  top: 0;
```

Comment:
I think this file can be a CSS module, nextjs has native support for CSS module, replace the file name with `modal.module.css` 

usage example
``` javascript
import styles from './layout.module.css';

export default function Layout({ children }) {
  return <div className={styles.container}>{children}</div>;
}